src/rules/maxLineLengthRule.ts

```diff
@@ -45,19 +79,31 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 
     public isEnabled(): boolean {
-        return super.isEnabled() && this.ruleArguments[0] as number > 0;
+        const argument: any = this.ruleArguments[0];
```

Comment:
you can omit the `: any` and `: boolean` type annotations.

Reply:
Though linter is complaining about infererred type